Microsoft.ML.sln

```diff
@@ -139,7 +139,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.DnnImageFeatur
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.DnnImageFeaturizer.ResNet101", "src\Microsoft.ML.DnnImageFeaturizer.ResNet101\Microsoft.ML.DnnImageFeaturizer.ResNet101.csproj", "{DB7CEB5E-8BE6-48A7-87BE-B91D9AE96F71}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.ML.Entrypoints", "src\Microsoft.ML.Entrypoints\Microsoft.ML.Entrypoints.csproj", "{7504D46F-E4B3-43CB-9B1C-82F3131F1C99}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ML.EntryPoints", "src\Microsoft.ML.Entrypoints\Microsoft.ML.EntryPoints.csproj", "{7504D46F-E4B3-43CB-9B1C-82F3131F1C99}"
```

Comment:
The folder still has the wrong casing

Reply:
Thanks.  Did the old `git mv foo foo-temp & git mv foo-temp Foo` to force git to pick up the case-sensitive renaming.

---
In reply to: [240847824](https://github.com/dotnet/machinelearning/pull/1859#discussion_r240847824) [](ancestors = 240847824)